contents/docs/surveys/_snippets/flutter-surveys-installation.mdx

```diff
@@ -0,0 +1,83 @@
+## Step one: Add PostHog to your app
+
+> 🚧 **Note:** Mobile surveys for Flutter is currently in **beta** and only available for **iOS**. We'd love to hear your feedback on betas via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asurveys%3Alow%3Atrue) or one of our other [support options](/docs/support-options).
+>
+> Using it requires PostHog's Flutter SDK version >= 5.1.0, but it's recommended to always use the latest version.
+
+import FlutterInstall from "../../integrate/_snippets/install-flutter.mdx"
+
+<FlutterInstall />
+
+## Step two: Enable surveys in your configuration
+
+To enable surveys in your Flutter app, update your PostHog configuration:
+
+```dart
+final config = PostHogConfig('<ph_project_api_key>');
+config.host = 'https://us.i.posthog.com';
+config.surveys = true; // Enable surveys (iOS Only)
+
+await Posthog().setup(config);
+```
+
+## Step three: Install PosthogObserver
+
+For surveys to display properly, you need to add the `PosthogObserver` to your app. The observer allows PostHog to determine the appropriate context for displaying surveys.
```

Comment:
```suggestion
For surveys to be shown, you need to add the `PosthogObserver` to your app. The observer allows PostHog to determine the appropriate context for displaying surveys.
```

i think `properly` is misleading here since its mandatory having the PosthogObserver